galaxy/templates/deployment-job.yaml

```diff
@@ -43,7 +43,15 @@ spec:
               mountPath: /galaxy/server/config/mutable/
               subPath: config
             - name: galaxy-data
-              mountPath: {{.Values.persistence.mountPath}}
+              mountPath: {{ .Values.persistence.mountPath }}
+        {{- if .Values.extraInitContainers -}}
+        {{- range $each := .Values.extraInitContainers -}}
+        {{- if $each.applyToJob -}}
+        {{- print "- " | nindent 8 -}}
+        {{- tpl ((unset (unset $each "applyToJob") "applyToWeb") | toYaml | indent 10 | trim) $ -}}
```

Comment:
This is great!

Reply:
It took a while to figure out, but I was so happy when it worked haha